### ISSUE TITLE
Screen reader announcement missing for when "pages are turned" (left/right icon buttons or arrow keys), whether in CSS columns paginated mode or scrolling

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#3105]

**Comment by danielweck** [comment]
Fixed via da8e2a5b56967c421cc471d02978bf2581c6300e and Electron v37
